src/components/CardComponents/CompletedButton.tsx

```diff
@@ -1,24 +1,48 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
 import { setResourceIsCompleted } from '../../store/resourcesSlice';
-import RadioButtonUncheckedIcon from '@material-ui/icons/RadioButtonUnchecked';
-import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import { withStyles } from '@material-ui/core/styles';
+import { styles } from './CardButtonStyles';
+import { SvgIcon } from '@material-ui/core';
+import { ReactComponent as CheckedButton} from './checkedButton.svg';
+import { ReactComponent as UncheckedButton} from './uncheckedButton.svg';
+
 
 interface CompletedButtonProps {
     id: number;
     isCompleted: boolean;
+    classes: any;
 }
 
 function CompletedButton(props: CompletedButtonProps) {
+    const { classes } = props;
     const dispatch = useDispatch();
-    return(<div><button onClick={() => dispatch(
-        setResourceIsCompleted({
-            id: props.id,
-            isCompleted: !props.isCompleted,
-        })
-    )}>
-        {props.isCompleted ? <RadioButtonUncheckedIcon /> : <FiberManualRecordIcon />}
-    </button></div>)
+    function handleOverlay(event) {
+        event.preventDefault();
+        // event.stopPropagation();
```

Comment:
Ooh could we either include this or remove it? 

(I think it may be useful to keep the stopPropagation (even though it doesn't necessarily affect it we do want to avoid it from being clicked twice in case future modifications))

Reply:
sounds good, kept it!